### DISCUSSION TITLE
Problem when updating podman from 4.4.1 to 4.4.4 on windows 11

### DISCUSSION THREAD

**Discussion by angelotrivelli** [containers/podman#18010] [Q&A]
A while ago I had installed **podman version 4.4.1** on my windows 11 PC.  It worked fine.

Today, however, I noticed that **podman 4.4.4** is the current version. I went to the github releases page and downloaded and ran the msi for version **4.4.4**.

That seemed to be OK. I rebooted the whole machine and then launched powershell...
```
PowerShell 7.3.3
PS C:\Users\angelo> podman --version
podman.exe version 4.4.4
```
OK, good, I did indeed update to **4.4.4**. And just checking my wsl vm's, **podman-machine-default** is `stopped`, as expected after a reboot:
```
PS C:\Users\angelo> wsl -l -v --all
  NAME                      STATE           VERSION
* Ubuntu                    Stopped         2
  podman-machine-default    Stopped         2

```

At this point, I figured, maybe I need to init the machine? It won't hurt, right?
```
PS C:\Users\angelo> podman machine init
Error: podman-machine-default: VM already exists
```
Well I did have **4.4.1** running before. So yeah, I suppose the VM *does* already exist and perhaps the msi installer merely updated it? So now I start the machine:
```
PS C:\Users\angelo> podman machine start
Starting machine "podman-machine-default"

This machine is currently configured in rootless mode. If your containers
require root permissions (e.g. ports < 1024), or if you run into compatibility
issues with non-podman clients, you can switch using the following command:

        podman machine set --rootful

API forwarding listening on: npipe:////./pipe/docker_engine

Docker API clients default to this address. You do not need to set DOCKER_HOST.
Machine **podman-machine-default** started successfully
```
... so far so good, that looks normal. 
But now when I go into the **podman-machine-default**'s shell and check the version:
```
You will be automatically entered into a nested process namespace where
systemd is running. If you need to access the parent namespace, hit ctrl-d
or type exit. This also means to log out you need to exit twice.

[user@thelaptop ~]$ podman --version
podman version 4.4.1
```

That's not good!

I get **version 4.4.4** when I run `podman --version` in powershell and **version 4.4.1** when I run `podman --version` on the **podman-machine-default** VM. 

Perhaps "it works", but it makes me super uncomfortable. There doesn't seem to be info on *updating* podman in the installation instructions. Did I miss something? Do have to totally uninstall podman and then re-install?

**Comment by Luap99** [comment]
see https://github.com/containers/podman/issues/17983
This is expected as the image is just a normal fedora version, you need to wait until the update lands in fedora.

**Comment by angelotrivelli** [comment]
Doing a "podman machine rm" and then "podman machine init" fixed the problem.

I was just confused about what a podman update would actually do.
